sdks/python/setup.py

```diff
@@ -153,6 +152,7 @@ def get_version():
     'pytz>=2018.3',
     'requests>=2.24.0,<3.0.0',
     'typing-extensions>=3.7.0,<4',
+    'pip-tools>=4.0.0'
```

Comment:
will provide a proper bound on the version for pip-tools. This is a placeholder for now